### PR TITLE
Fixed_alpha_shape_3: Replace destroyed with swapped in documentation

### PR DESCRIPTION
**Pull request by afabri**
## Summary of Changes

Rephrase the documentation of a  constructor.

## Release Management

* Affected package(s): Alpha_shapes_3
* Issue(s) solved (if any):  See [stackoverflow](https://stackoverflow.com/questions/57572583/does-fixed-alpha-shape-3-destroy-or-modify-original-triangulation)

### CODE REVIEW

File: Alpha_shapes_3/doc/Alpha_shapes_3/CGAL/Alpha_shape_cell_base_3.h [hunk 1]
```diff
@@ -107,28 +107,28 @@ void set_alpha_max(NT alpha);
...
 /*!
-Returns true for convex hull faces. 
+returns `true` for convex hull faces. 
 */ 
 bool is_on_chull() const; 
 
 /*!
-Returns the `alpha_min`. 
-\pre `is_Gabriel()` returns false; 
+returns the `alpha_min`.
```
Comment (MaelRL) [inline Alpha_shapes_3/doc/Alpha_shapes_3/CGAL/Alpha_shape_cell_base_3.h]:
```suggestion
returns `alpha_min`. 
```

File: Alpha_shapes_3/doc/Alpha_shapes_3/CGAL/Fixed_alpha_shape_3.h [hunk 2]
```diff
@@ -55,11 +55,11 @@ Enum to classify the simplices of the underlying
 triangulation with respect to a given alpha value. 
 
 Each k-dimensional simplex of the triangulation 
-can be classified as EXTERIOR, SINGULAR, REGULAR 
-or INTERIOR. 
-A \f$ k\f$ simplex is REGULAR if it is on the boundary 
+can be classified as `EXTERIOR`, `SINGULAR`, `REGULAR`
```
Comment (MaelRL) [inline Alpha_shapes_3/doc/Alpha_shapes_3/CGAL/Fixed_alpha_shape_3.h]:
```suggestion
can be classified as `EXTERIOR`, `SINGULAR`, `REGULAR`,
```

File: Alpha_shapes_3/doc/Alpha_shapes_3/CGAL/Fixed_alpha_shape_3.h [hunk 3]
```diff
@@ -198,13 +198,13 @@ OutputIterator get_alpha_shape_vertices(OutputIterator it, Classification_type t
...
 /*!
-Inserts the fixed alpha shape `A` into the stream `os`. 
+inserts the fixed alpha shape `A` into the stream `os`. 
 
 
 An overlaoad of `operator<<` must be available for `GT::Point`.
 \relates Fixed_alpha_shape_3 
 */ 
-ostream& operator<<(ostream& os, const Fixed_alpha_shape_3<Dt>& A); 
+  std::ostream& operator<<(std::ostream& os, const Fixed_alpha_shape_3<Dt>& A);
```
Comment (MaelRL) [inline Alpha_shapes_3/doc/Alpha_shapes_3/CGAL/Fixed_alpha_shape_3.h]:
```suggestion
std::ostream& operator<<(std::ostream& os, const Fixed_alpha_shape_3<Dt>& A); 
```

**Comment by maxGimeno** [comment]
Successfully tested i nhttps://cgal.geometryfactory.com/CGAL/testsuite/results-5.0-Ic-118.shtml

**Pull request merged**